modules/xmpp/RtxModifier.spec.js

```diff
@@ -0,0 +1,268 @@
+/*eslint-disable max-len*/
```

Comment:
Why do we have mixed requires with imports and vars with const/let in this file?